docs/concept-exercises.md

```diff
@@ -51,30 +51,51 @@ An exercise has the following files. In the browser they will show at the releva
+**Purpose:** Provide instructions for the exercise.
+
+This file is split into two halves. 
+1. The first half explains the "story" or "theme" of the exericise. It should generally contain no code samples.
+2. The second half provides clear instructions of what a student needs to do, providing tasks.
+
+Each tasks must conform to the following standard:
+- Start with a third-level heading starting with a number (.e.g `### 1.`, `### 2.`)
+- Provide a method that a student needs to define with its signature(e.g. `X(...)` that takes an A and returns a Z),
+- Provide at example usage of that function. These examples should be different examples to those given in the tests.
 
-See the C# floating-point-numbers exercise's [instructions.md file][csharp-docs-instructions.md] for an example.
+Avoid any controversial topics in the stories - be empathetic to triggers that people might have - there's normally a less controversial or risque variant of a story that can be used.
+
+See the C# numbers exercise's [instructions.md file][csharp-docs-instructions.md] for an example. Notice the clear distinction between the story at the top and the tasks with code samples below.
 
 ### `.docs/hints.md`
 
-If the student gets stuck, we will allow them to click a button requesting a hint, which shows this file. This will not be a "recommended" path and we will (softly) discourage them using it unless they can't progress without it. As such, it's worth considering that the student reading it will be a little confused/overwhelmed and maybe frustrated.
+**Purpose:** Provide hints to a student to help them get themselves unstuck n an exercise.
 
-The file should contain both general and task-specific "hints". These hints should be enough to unblock almost any student. They might link to the docs of the functions that need to be used.
+- If the student gets stuck, we will allow them to click a button requesting a hint, which will show the relevant part of file. 
+- Hints should be bullet-pointed underneath headings.
+- General hints about the exericse can appear under the`### General` heading.
+- Task-specific hints should appear undernearth headings that match their task in the `instructions.md` (e.g. `### 2.`).
 
-The hints should not spell out the solution, but instead point to a resource describing the solution (e.g. linking to documentation for the function to use).
+Viewing hints will not be a "recommended" path and we will (softly) discourage them using it unless they can't progress without it. As such, it's worth considering that the student reading it will be a little confused/overwhelmed and maybe frustrated. These hints should be enough to unblock almost any student. They might link to the docs of the functions that need to be used. The hints should not spell out the solution, but instead point to a resource describing the solution (e.g. linking to documentation for the function to use).
```

Comment:
```suggestion
Viewing hints will not be a "recommended" path and we will (softly) discourage using it unless the student can't progress without it. As such, it's worth considering that the student reading it will be a little confused/overwhelmed and maybe frustrated. These hints should be enough to unblock almost any student. They might link to the docs of the functions that need to be used. The hints should not spell out the solution, but instead point to a resource describing the solution (e.g. linking to documentation for the function to use).
```